src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -661,6 +661,13 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         // Discover translation directories
         $dirs = array();
+
+        foreach ($config['paths'] as $path) {
+            if (is_dir($path)) {
+                $dirs[] = $path;
+            }
```

Comment:
Technically, that could still become an issue if a parameter were changed in a compiler pass, couldn't it?